css/inner-page.css

```diff
@@ -142,6 +142,15 @@ h4 code.highlighter-rouge {
     margin-top: 7px;
 }
 
+.post-content h4[id] {
+    font-family: "Space Grotesk",
+        sans-serif;
```

Comment:
Unless there’s a particular reason for this, leave on same line with the rest of the declaration.

Reply:
@j9t  fixed